frontend/providers/applaunchpad/deploy/manifests/deploy.yaml.tmpl

```diff
@@ -32,6 +32,10 @@ spec:
         app: applaunchpad-frontend
     spec:
       containers:
+        - env:
+            - name: SEALOS_DOMAIN
+              value: {{ .cloudDomain }}
+            - name: FASTGPT_KEY
```

Comment:
value？

Reply:
can be empty without value？

Reply:
In k8s standard, env's key can stay without value. But can applaunchpad start without this env? @c121914yu

Reply:
> In k8s standard, env's key can stay without value. But can applaunchpad start without this env? @c121914yu

yes. It can be started normally. This value is only required when using event analysis